src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ReleaseFileServiceTests.cs

```diff
@@ -374,11 +372,37 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             blobStorageService
                 .SetupDownloadToStream(PublicReleaseFiles, releaseFile2.PublicPath(), "Test ancillary blob");
 
+            var subjectIds = releaseFiles
+                .Where(rf => rf.File.SubjectId.HasValue)
+                .Select(rf => rf.File.SubjectId.GetValueOrDefault())
+                .ToList();
+
+            var dataGuidanceFileWriter = new Mock<IDataGuidanceFileWriter>(MockBehavior.Strict);
+
+            dataGuidanceFileWriter
+                .Setup(
+                    s => s.WriteToStream(
+                        It.IsAny<Stream>(),
+                        It.Is<Release>(r => r.Id == release.Id),
+                        It.Is<IEnumerable<Guid>>(
+                            ids => ids.All(id => subjectIds.Contains(id))
```

Comment:
A shame that there's not an `It` method that does this for us already.  Also a shame that we can't create extension methods on static classes or we could have created one for reuse!

Reply:
Yeh this kinda sucked. Moq has its nice parts, but it's also got a number of rough edges for stuff like this